### PR TITLE
core: adding 'radosgw-admin' command support

### LINKED ISSUE #253: Whither radosgw-admin?
This plugin is good stuff.  I find however that many of the tasks I invoke a toolbox exec for are "radosgw" commands, which are currently not implemented.  So consider this an RFE.

I also often do "watch ceph -s".  I haven't tested, but I wonder what the latency of krew commands is like, if this would work well with the default 2 second interval, or if perhaps the plugin running the watch itself might work well.

### PR DESCRIPTION
**Pull request by subhamkrai**
adding support for 'radosgw-admin' command in the cli tool and also adding docs and ci.

**Issue resolved by this Pull Request:**
Resolves #253 

**Checklist:**

- [ ] **Commit Message Formatting**: Commit titles and messages follow guidelines in the [developer guide](https://rook.io/docs/rook/latest/Contributing/development-flow/#commit-structure).
- [ ] Reviewed the developer guide on [Submitting a Pull Request](https://rook.io/docs/rook/latest/Contributing/development-flow/#submitting-a-pull-request)
- [ ] Documentation has been updated, if necessary.
- [ ] Unit tests have been added, if necessary.
- [ ] Integration tests have been added, if necessary.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,6 +58,8 @@ These are args currently supported:
 
 - `rados <args>` : Run a Rados CLI command. Supports any arguments the `rados` command supports. See [Rados docs](https://docs.ceph.com/en/latest/man/8/rados/) for more.
 
+- `radosgw-admin <args>` : Run a Radosgw CLI command. Supports any arguments the `radosgw-admin` command supports. See [Radosgw-admin docs](https://docs.ceph.com/en/latest/man/8/radosgw-admin/) for more.
```
Comment (travisn) [inline README.md:61]:
```suggestion
- `radosgw-admin <args>` : Run an RGW CLI command. Supports any arguments the `radosgw-admin` command supports. See the [radosgw-admin docs](https://docs.ceph.com/en/latest/man/8/radosgw-admin/) for more.
```

File: cmd/commands/radosgw.go [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+import (
+	"github.com/rook/kubectl-rook-ceph/pkg/exec"
+	"github.com/rook/kubectl-rook-ceph/pkg/logging"
+	"github.com/spf13/cobra"
+)
+
+// RadosgwCmd represents the radosgw command
+var RadosgwCmd = &cobra.Command{
+	Use:                "radosgw-admin",
+	Short:              "call a 'radosgw-admin' CLI command with arbitrary args",
```
Comment (travisn) [inline cmd/commands/radosgw.go:29]:
```suggestion
	Short:              "call a 'radosgw-admin' CLI command",
```

File: docs/radosgw-admin.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
+# Radosgw-admin
```
Comment (travisn) [inline docs/radosgw-admin.md:1]:
```suggestion
# radosgw-admin
```

File: docs/radosgw-admin.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+This used to run any radosgw-admin cli command with with arbitrary args.
+
+## Examples
+
+```bash
+kubectl rook-ceph radosgw-admin user create --display-name="johnny rotten" --uid=johnny
```
Comment (travisn) [inline docs/radosgw-admin.md:8]: rotten? This suggestion might be boring though :)
```suggestion
kubectl rook-ceph radosgw-admin user create --display-name="my user" --uid=myuser
```

File: docs/radosgw-admin.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+
+# Info: running 'radosgw-admin' command with args: [user create --display-name=johnny rotten --uid=johnny]
+# {
```
Comment (travisn) [inline docs/radosgw-admin.md:11]: How about not include the full example output so we don't have to maintain it?
```suggestion
...
```

File: .github/workflows/go-test.yaml [hunk 6]
```diff
@@ -67,6 +67,11 @@ jobs:
           set -ex
           kubectl rook-ceph --context=$(kubectl config current-context) rados df
 
+      - name: Radosgw-admin create user
```
Comment (travisn) [inline .github/workflows/go-test.yaml:70]: Let's use the lower case name of the tool everywhere
```suggestion
      - name: radosgw-admin create user
```

**Review by travisn — CHANGES_REQUESTED** [review]
Mainly some small doc suggestions

File: README.md (hunk 1 above)
Comment (subhamkrai) [reply README.md]: done

File: docs/radosgw-admin.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
+# radosgw-admin
+
+This used to run any radosgw-admin cli command with with arbitrary args.
```
Comment (travisn) [inline docs/radosgw-admin.md:3]:
```suggestion
This runs any radosgw-admin CLI command with arbitrary args.
```

File: docs/radosgw-admin.md [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+
+## Examples
+
+```bash
+kubectl rook-ceph radosgw-admin user create --display-name="my user" --uid=myuser
+
+# Info: running 'radosgw-admin' command with args: [user create --display-name=johnny rotten --uid=johnny]
```
Comment (travisn) [inline docs/radosgw-admin.md:10]: The output still has jonny rotten :)

File: docs/radosgw-admin.md [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+# {
+#     "user_id": "johnny",
+#     "display_name": "johnny rotten",
+#     "email": "",
```
Comment (travisn) [inline docs/radosgw-admin.md:14]: I'd suggest we keep the output even simpler and remove lines 14-30.

**Review by travisn — CHANGES_REQUESTED** [review]

File: docs/radosgw-admin.md (hunk 8 above)
Comment (subhamkrai) [reply docs/radosgw-admin.md]: I just changed the name in the command missed updating the output

**Review by travisn — APPROVED** [review]

**Pull request merged**